tests/lora/test_lora_checkpoints.py

```diff
@@ -28,6 +28,7 @@ def test_load_checkpoints(
         # and the test should pass.
         LoRAModel.from_local_checkpoint(
             baichuan_lora_files,
+            4096,
```

Comment:
use kwarg instead

Reply:
Default None == meaning it will use scaling factor 1. Also added docstring